iml-gui/crate/src/page/fs_dashboard.rs

```diff
@@ -0,0 +1,17 @@
+use crate::Msg;
+use iml_wire_types::warp_drive::ArcCache;
+use seed::{prelude::*, *};
+
+#[derive(Default)]
+pub struct Model {
+    pub fs_name: String,
+}
+
+pub fn view(_: &ArcCache, model: &Model) -> impl View<Msg> {
+    iframe![attrs! {
+        At::Src => format!("https://localhost:7444/grafana/d/2vkrIIQWz/fs-dashboard?var-fs_name={}&orgId=1&from=now-15m&to=now&kiosk", model.fs_name),
```

Comment:
```suggestion
        At::Src => format!("/grafana/d/2vkrIIQWz/fs-dashboard?var-fs_name={}&orgId=1&from=now-15m&to=now&kiosk", model.fs_name),
```

We can use webpack proxy to avoid hardcoding urls.

ref:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/5a0d06eeeb6e20024d4b560af964043b4629d5ae/iml-gui/configs/webpack.config.js#L39